### PR TITLE
support leaky_relu add test for inplace relu

### PR DESCRIPTION
**Pull request by eaidova**

### CODE REVIEW

File: tests/layer_tests/pytorch_tests/pytorch_layer_test_class.py [hunk 1]
```diff
@@ -85,6 +85,9 @@ def _test(self, model, ref_net, kind, ie_device, precision, ir_version, infer_ti
 
         if not isinstance(fw_res, tuple):
             fw_res = (fw_res,)
+        
+        if inplace:
+            fw_res = tuple([fw_res[0], fw_res[0]])
```
Comment (mvafin) [inline tests/layer_tests/pytorch_tests/pytorch_layer_test_class.py]: I think this shouldn't be correct way of handling inplace ops. The better way should be making model that has 2 outputs: `x` and `y` (if we consider activation like `y=relu(x)`)

File: tests/layer_tests/pytorch_tests/test_leaky_relu.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+            def forward(self, x):
+                return F.leaky_relu(x, self.alpha, inplace=self.inplace)
+            
+        ref_net = None
+
+        return aten_leaky_relu(alpha, inplace), ref_net, "aten::leaky_relu" if not inplace else "aten::leaky_relu_"
+
+
+    @pytest.mark.parametrize("alpha,inplace", [(0.01, True), (0.01, False), (-0.01, True), (-0.01, False)])
```
Comment (mvafin) [inline tests/layer_tests/pytorch_tests/test_leaky_relu.py]: `0.01` is default value for `leaky_relu`, could you add non default positive case?

Comment (eaidova) [reply tests/layer_tests/pytorch_tests/test_leaky_relu.py]: added

File: tests/layer_tests/pytorch_tests/pytorch_layer_test_class.py (hunk 1 above)
Comment (eaidova) [reply tests/layer_tests/pytorch_tests/pytorch_layer_test_class.py]: updated

File: tests/layer_tests/pytorch_tests/pytorch_layer_test_class.py [hunk 3]
```diff
@@ -115,7 +116,7 @@ def _prepare_input(self):
         raise RuntimeError("Please provide inputs generation function")
 
 
-def get_params(ie_device=None, precision=None):
+def get_params(ie_device=['CPU'], precision=None):
```
Comment (mvafin) [inline tests/layer_tests/pytorch_tests/pytorch_layer_test_class.py]: please remove this change. We would want to test GPU if we at some point get into CI.

Comment (eaidova) [reply tests/layer_tests/pytorch_tests/pytorch_layer_test_class.py]: sorry, accidentally commited

**Review by mvafin — APPROVED** [review]

**Pull request merged**